src/App.jsx

```diff
@@ -1,27 +1,72 @@
+import { setUtahHeaderSettings } from '@utahdts/utah-design-system-header';
+import { useEffect } from 'react';
 import { useRemoteConfigString } from 'reactfire';
-import './App.css';
-import { fieldNames } from './config';
+import MapComponent from './components/Map.jsx';
+import SearchWizard from './components/search-wizard/Wizard.jsx';
 
 function App() {
   const queryLayersConfig = useRemoteConfigString('queryLayers');
+  const linksConfig = useRemoteConfigString('links');
+
+  useEffect(() => {
+    if (linksConfig.status !== 'success') {
+      return;
+    }
+
+    const links = JSON.parse(linksConfig.data);
+    setUtahHeaderSettings({
+      // this prop's implementation has not been released yet
+      // domLocationTarget: {
+      //   element: headerRef.current,
+      // },
+      title: 'Utah Environmental Interactive Map',
+      titleURL: null,
+      logo: '<img src="/deq_logo.png" alt="DEQ Logo" />',
+      utahId: false,
+      actionItems: [
+        {
+          actionPopupMenu: {
+            menuItems: [
+              {
+                actionUrl: {
+                  url: links['2'].url,
+                  openInNewTab: true,
+                },
+                title: links['2'].description,
+              },
+              {
+                // TODO: add a modal for this
+                actionFunction: () => window.alert('not implemented yet!'),
+                title: 'Disclaimer',
+              },
+            ],
+            title: 'Links',
+          },
+          className: 'icon-waffle',
+          showTitle: false,
+          title: 'Links',
+          icon: '<span class="utds-icon-before-waffle" aria-hidden="true" />',
+        },
+      ],
+    });
+  }, [linksConfig]);
 
   if (queryLayersConfig.status === 'loading') {
     console.log('loading remote config');
 
-    return <span>getting configs...</span>;
+    return null;
```

Comment:
**suggestion**: consider using react-content-loader to show a loading skeleton

https://designsystem.dev.utah.gov/resources/mockups#section-spinners